packages/driver/cypress/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": ["es5", "dom"],
+    "target": "es2020",
+    "lib": ["es2020", "dom"],
```

Comment:
We had some tests that was testing bigint? I think? But this isn't usable until es2020.